src/buildHTML.js

```diff
@@ -1401,10 +1401,29 @@ groupTypes.accent = function(group, options) {
         // thus shows up much too far to the left. To account for this, we add a
         // specific class which shifts the accent over to where we want it.
         // TODO(emily): Fix this in a better way, like by changing the font
-        const vecClass = group.value.label === "\\vec" ? "accent-vec" : null;
+        // Similarly, all text accents are combining characters and require
+        // a different adjustment.
+        let accentClass = null;
+        if (group.value.label === "\\vec") {
+            accentClass = "accent-vec";
+        } else if (utils.contains(["\\'", "\\`", "\\^", "\\~", "\\=", "\\u",
+            "\\.", '\\"', "\\r", "\\H", "\\v"], group.value.label)) {
+            accentClass = "accent-text";
+        }
 
-        accentBody = makeSpan(["accent-body", vecClass], [
-            makeSpan([], [accent])]);
+        // Some browsers (Safari in particular) don't like combining
+        // characters without a preceding character.  For each such accent,
+        // we add an artificial nonbreaking space with a negative margin
+        // to compensate for its width.
```

Comment:
It's probably worth leaving this code in here with a TODO describe what's necessary to get the accents lining up on Chrome that way other contributors can build on top of the foundation you started here.